pcsx2/HwRead.cpp

```diff
@@ -42,11 +44,16 @@ mem32_t __fastcall _hwRead32(u32 mem)
 {
 	pxAssume( (mem & 0x03) == 0 );
 
+	// todo: psx mode: this is new
```

Comment:
code need to be moved in the switch page section below. (need to find the good page, I think it is address >> 16 but I'm not sure)